src/motor-interface-pru.p

```diff
@@ -245,9 +276,11 @@ DONE_STEP_GEN:
 
 	;; Next position in ring buffer
 	ADD r2, r2, QUEUE_ELEMENT_SIZE
+	ADD r28.b3, r28.b3, 1 ; add + 1 to the MSB byte
```

Comment:
I'd probably move the ';' comment more out for readability; so more space before the ';'.